packages/plugin-core/src/features/HelpFeedbackTreeview.ts

```diff
@@ -0,0 +1,127 @@
+  ThemeIcon,
+  TreeDataProvider,
+  TreeItem,
+  TreeItemCollapsibleState,
+} from "vscode";
+import * as vscode from "vscode";
+import {
+  assertUnreachable,
+  DendronTreeViewKey,
+  VSCodeEvents,
+} from "@dendronhq/common-all";
+import { AnalyticsUtils } from "../utils/analytics";
+
+enum MenuItem {
+  getStarted = "Get Started",
+  readDocs = "Read Documentation",
+  reviewIssues = "Review Issues",
+  reportIssue = "Report Issue",
+  joinCommunity = "Join our Community!",
+}
+
+class HelpFeedbackTreeDataProvider implements TreeDataProvider<MenuItem> {
+  ALL_ITEMS = Object.values(MenuItem) as MenuItem[];
+
+  onDidChangeTreeData?: Event<void | MenuItem | null | undefined> | undefined;
+
+  getTreeItem(element: MenuItem): TreeItem {
+    let iconPath: vscode.ThemeIcon;
+
+    switch (element) {
+      case MenuItem.getStarted:
+        iconPath = new ThemeIcon("star");
+        break;
+
+      case MenuItem.readDocs:
+        iconPath = new ThemeIcon("book");
+        break;
+
+      case MenuItem.reviewIssues:
+        iconPath = new ThemeIcon("issues");
+        break;
+
+      case MenuItem.reportIssue:
+        iconPath = new ThemeIcon("comment");
+        break;
+
+      case MenuItem.joinCommunity:
+        iconPath = new ThemeIcon("organization");
+        break;
+
+      default:
+        assertUnreachable(element);
+    }
+
+    return {
+      label: element.toString(),
+      collapsibleState: TreeItemCollapsibleState.None,
+      iconPath,
+    };
+  }
+  getChildren(element?: MenuItem): ProviderResult<MenuItem[]> {
+    switch (element) {
+      case undefined:
+        return this.ALL_ITEMS;
+      default:
+        return [];
+    }
+  }
+}
+
+function openUrl(url: string) {
+  vscode.commands.executeCommand("vscode.open", url);
+}
+
+/**
+ * Creates a tree view for the basic 'Help and Feedback' panel in the Dendron
+ * Custom View Container
+ * @returns
+ */
+export default function setupHelpFeedbackTreeView(): vscode.TreeView<MenuItem> {
+  const treeView = vscode.window.createTreeView(
+    DendronTreeViewKey.HELP_AND_FEEDBACK,
+    {
+      treeDataProvider: new HelpFeedbackTreeDataProvider(),
+    }
+  );
+
+  treeView.onDidChangeSelection((e) => {
+    const item = e.selection[0];
+
+    AnalyticsUtils.track(VSCodeEvents.HelpAndFeedbackItemClicked, {
+      menuItem: item,
+    });
+
+    switch (item) {
+      case MenuItem.getStarted:
+        openUrl(
+          "https://wiki.dendron.so/notes/678c77d9-ef2c-4537-97b5-64556d6337f1/"
+        );
+        break;
+
+      case MenuItem.readDocs:
+        openUrl("https://wiki.dendron.so/notes/F5ly88oU0ecUiACLxET5E/");
+        break;
+
+      case MenuItem.reviewIssues:
+        openUrl("https://github.com/dendronhq/dendron/issues");
+        break;
+
+      case MenuItem.reportIssue:
+        openUrl("https://github.com/dendronhq/dendron/issues/new/choose");
+        break;
+
+      case MenuItem.joinCommunity:
+        openUrl("https://discord.gg/S5MQrwZD7R");
```

Comment:
@kevinslin - can you get me a Discord link? Right now, it'll say "invited by mobius" - do we have a general account or something like that?

Reply:
see https://link.dendron.so/discord

also, frequently used links are kept in [[Common Links|dendron://dendron.handbook/area.community.ref.links]]